ember/.stylelintrc.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+module.exports = {
+  extends: ["stylelint-config-standard", "stylelint-prettier/recommended"],
+  rules: {
+    "import-notation": null,
+    "at-rule-no-unknown": null,
+  },
```

Comment:
```suggestion
 extends: ["stylelint-config-standard-scss", "stylelint-prettier/recommended"],
```
 
Please extend from those configs as already mentioned. The extra rules should not be necessary.
And install the packages via `yarn add -D stylelint-config-standard-scss stylelint-scss`.

Reply:
And remove `stylelint-config-standard` from `package.json`.